### ISSUE TITLE
Systray-Indicator is not a solution

### ISSUE THREAD

**Issue by ghost** [MichelMichels/cryptocoin-indicator#1]
Hi. 

Please see, systray-indicator was merged inside cinnamon. If you want to used the indicators in cinnamon, you will need to enable it in the cinnamon-settings inside the general settings. The extension only break cinnamon currently.

Regards.

**Comment by MichelMichels** [comment]
So you are saying this has native support in cinnamon? I will try to look at this asap.

**Comment by ghost** [comment]
Yes... I was who merged it... And is exactly what the applet provided, so the applet conflicted with the current cinnamon code.
